pkg/apis/numaflow/v1alpha1/serving_store.go

```diff
@@ -27,10 +27,21 @@ type ServingStore struct {
 	Container *Container `json:"container" protobuf:"bytes,2,opt,name=container"`
 }
 
+// volume mount to the runtime path
```

Comment:
It will be removed in https://github.com/numaproj/numaflow/pull/2468, leave this file untouched,  to make the merge easier.